libgalois/include/katana/DynamicBitset.h

```diff
@@ -106,6 +109,7 @@ class KATANA_EXPORT DynamicBitset {
    */
   void clear() {
     num_bits_ = 0;
+    std::fill(bitvec_.begin(), bitvec_.end(), 0);
```

Comment:
nit: It's better to move this into a private function and we may want to parallelize this in the future. You can add a TODO on me for that. Something like:
`\\ TODO(roshan) parallelize this`